README.md

```diff
@@ -64,7 +56,8 @@ bin/plugin install --no-verify
 ```
 - Run Logstash with your plugin
 ```sh
-bin/logstash -e 'filter {awesome {}}'
+bin/logstash -e \
+    'input { generator { count => 100 } } output { dynatrace { api_key => "your_api_key_here" active_gate_url => "https://abc12345.live.dynatrace.com/api/v2/logs/ingest" } }'
```

Comment:
```suggestion
    'input { generator { count => 100 } } output { dynatrace { api_key => "your_api_key_here" ingest_endpoint_url => "https://{your-environment-id}.live.dynatrace.com/api/v2/logs/ingest" } }'
```
Will the generator work without specifying a message?

Adding the placeholder in the URL makes clear that we don't provide a demo endpoint for evaluation here but that one will have to add their actual ID as it will throw with that URL.

Reply:
>Will the generator work without specifying a message?

yes. it just sends sequentially numbered messages